source/uwp/Renderer/lib/AdaptiveCard.cpp

```diff
@@ -113,7 +113,7 @@ namespace AdaptiveNamespace
         }
         catch (const AdaptiveCardParseException& e)
         {
-            ComPtr<IVector<IAdaptiveError*>> errors;
+            ComPtr<IVector<ABI::AdaptiveNamespace::AdaptiveError*>> errors;
```

Comment:
>ABI::AdaptiveNamespace:: [](start = 27, length = 24)

Is there a particular reason we are explicit about ABI::AdpativeNamespace when we have the using directive for the namespace at the top of the file?

Reply:
It's to distinguish the runtimeclass ABI:AdaptiveNamespace::Foo from our implementation class AdaptiveNamespace::Foo (No ABI::).

---
In reply to: [249209912](https://github.com/Microsoft/AdaptiveCards/pull/2317#discussion_r249209912) [](ancestors = 249209912)